gulpfile.js

```diff
@@ -344,12 +426,18 @@ gulp.task('watch', ['build'], function watchTask() {
 // Set up a webserver for the static assets
 gulp.task('connect', function connectTask() {
   connect.server({
-    root: DEST_PATH,
+    root: DIST_PATH,
     livereload: false,
-    port: 9988
+    port: 8000
```

Comment:
Might want to put this at a different port than 8000, to avoid conflicting with Django in case you want to run both at the same time for comparison's sake. I think that's why I had it at a weird port